documents/documents.md

```diff
@@ -0,0 +1,65 @@
+## Document
+
+### Create a Document
+
+```rb
+Razorpay::Document.create({
```

Comment:
Can you add relevant public API doc for these APIs ?

Reply:
for document create : https://razorpay.com/docs/api/documents/create/